.gitignore

```diff
@@ -0,0 +1,5 @@
+.dart_tool/
+.packages
```

Comment:
And such changes are also not really needed

Reply:
It was added automatically by IntelliJ. I think it's quite ok to have them ignore as you don't want this to be committed, but I'll remove